tests/IntegrationTests/Tls.fs

```diff
@@ -11,44 +11,47 @@ open Pulsar.Client.Common
 open Pulsar.Client.IntegrationTests.Common
 
 
+let tlsTransport client testName =
+    testTask testName {

```

Comment:
Can you please add debug logs at the beginning and the end of the test (similar to other integration tests)